.github/workflows/jdl.yml

```diff
@@ -88,14 +94,35 @@ jobs:
       - name: 'SETUP: get date'
         id: get-date
         run: echo "::set-output name=date::$(/bin/date -u "+%Y%m%d")"
-      - name: 'SETUP: docker'
+
+      - name: 'SETUP Linux: Set up swap space'
+        if: runner.os == 'Linux'
+        # v1.0 (49819abfb41bd9b44fb781159c033dba90353a7c)
+        uses: pierotofy/set-swap-space@49819abfb41bd9b44fb781159c033dba90353a7c
+        with:
+          swap-size-gb: 10
+      - name:
+          'SETUP Windows: Configure pagefile'
```

Comment:
Linux uses "Set up swap space" then we use "Configure pagefile" here - I think it'd be better to be consistent.